Formula/a/aicommit.rb

```diff
@@ -0,0 +1,20 @@
+class Aicommit < Formula
+  desc "AI-powered commit message generator"
+  homepage "https://github.com/coder/aicommit"
+  url "https://github.com/coder/aicommit/archive/refs/tags/v0.6.3.tar.gz"
+  sha256 "f42fac51fbe334f4d4057622b152eff168f4aa28d6da484af1cea966abd836a1"
+  license "CC0-1.0"
+
+  depends_on "make" => :build
+  depends_on "go" => "1.21"
+
+  def install
+    ENV["VERSION"] = version
+    system "make", "build"
+    bin.install "bin/aicommit"
```

Comment:
Let's just call `go build` directly; it's a lot simpler.
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w -X main.Version=#{version}"), "./cmd/aicommit"
```

Reply:
I could but then I would be duplicating this logic between the brew and the Makefile which might lead to future bugs. If it's your preferred style to avoid system make I'll happily call go build directly.

Reply:
Yes, let's call `go build` directly please. We do it in several hundred other formulae, and experience shows that the likelihood of bugs being introduced this way is sufficiently small.